packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -238,24 +256,34 @@ const builtInFlags = [
         usage: '--stats <value>',
         type: [String, Boolean],
         negative: true,
+        help: 'base',
         description: 'It instructs webpack on how to treat the stats e.g. verbose',
         link: 'https://webpack.js.org/configuration/stats/#stats',
     },
     {
         name: 'watch',
         usage: '--watch',
         type: Boolean,
+        help: 'base',
         negative: true,
         alias: 'w',
         description: 'Watch for files changes',
         link: 'https://webpack.js.org/configuration/watch/',
     },
 ];
 
+// single word verbose options
+const verboseFlags = ['amd', 'node', 'externals', 'dependencies'];
+
 // Extract all the flags being exported from core.
 // A list of cli flags generated by core can be found here https://github.com/webpack/webpack/blob/master/test/__snapshots__/Cli.test.js.snap
 const coreFlags = cli
     ? Object.entries(cli.getArguments()).map(([flag, meta]) => {
+          if (flag.includes('-') || verboseFlags.includes(flag)) {
```

Comment:
Why we need `flag.includes('-')` and this complex logic? Just create `notVerboseFlags` array and put flags to simple help to this array

Reply:
This is not needed anymore. Old leftover 😞